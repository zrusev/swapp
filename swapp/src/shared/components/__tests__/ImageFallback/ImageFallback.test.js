import React from 'react';
import { shallow } from 'enzyme';

import ReactImageFallback from '../../ImageFallback/ImageFallback';

describe('should ReactImageFallback be styled', () => {
  it('should mount', () => {
    const props = {
      fallbackImage: '',
    };
    const wrapper = shallow(<ReactImageFallback {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
