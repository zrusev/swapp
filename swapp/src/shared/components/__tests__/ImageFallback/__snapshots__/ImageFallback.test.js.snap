// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should ReactImageFallback be styled should mount 1`] = `
<ReactImageFallback
  fallbackImage=""
  initialImage={null}
/>
`;
